### DISCUSSION TITLE
Possibility to delete records from the dataset after starting screening

### DISCUSSION THREAD

**Discussion by Woensvriend** [asreview/asreview#1741] [Q&A] [answered]
Dear ASReview community,

As recommended because ASReview can work suboptimally (e.g. #529), I deduplicated records in my dataset as much as possible. Unfortunately, after screening ~500 records, the AI presents one study twice in a row to me. (I guess it slipped through in the deduplication process because of its title spelling variations.) I've searched for a thread that concerns this, but cannot find anything. Is it possible to delete a record from the dataset, after having started screening? If not, I guess its better to label the second one as relevant instead of irrelevant, in order to 'limit the damage' as much as possible?

**Comment by Rensvandeschoot** [comment, marked as answer]
I would recommend applying the same label for all duplicate records so that the model gets proper labels and does not get confused. Removing duplicate records from the data during screening would be a nice feature! Could you start a feature request? https://github.com/asreview/asreview/issues/new/choose

**Reply by Woensvriend** [reply]
Thanks for the reply @Rensvandeschoot. Indeed, I didn't mention that I labeled the first one _relevant_. I will start a feature request soon!

**Reply by Woensvriend** [reply]
>I will start a feature request soon!

Started under #1759.
